tests/MongoDB.Driver.Core.Tests/Core/Clusters/DnsClientWrapperTests.cs

```diff
@@ -33,6 +33,27 @@ public void constructor_should_initialize_instance()
             subject.LookupClient.Should().NotBeNull();
         }
 
+        [Theory]
+        [InlineData("test", "test")]
+        [InlineData("test.test", "test.test")]
+        [InlineData("128.0.0", null)]
```

Comment:
Not clear why this IP address `127.0.0`(see the previous commit) is successfully processed on EG into `ip-127-0-0-0.ec2.internal`. Probably EG contains some specific configuration for that. Locally (with `vpn`) this address returns `null`. Anyway, the idea of the test was that not full IP address will return `null`, so the currently specified value proves it

Reply:
`128.0.0` looks like a typo.

But according to your comment it is intentional?

Perhaps add `// this host value is intentionally incomplete` to line 39

Reply:
>But according to your comment it is intentional?

yes
I don't have problems with `127.0.0` locally, but EG returns not expected result for me.

Probably the test name tells about it fully enough?

Reply:
Now that you say that... is the test name even correct?

The first two test values look like valid host names. Only the third one is not valid.